metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1445,6 +1445,51 @@ def run_assertions_for_asset(
 
         return res["runAssertionsForAsset"]
 
+    def get_entities_v2(
```

Comment:
So primary limitation here is ... 

the fact you can only retrieve 1 type of entity. okay. is this same as other get_entities endpoints we have?

Reply:
Yeah, I believe we do have an endpoint to retrieve one record. What we need to make it optimal here is to retrieve specific aspect for multiple entities of the same type, which is exactly what this API does.